core/src/main/java/org/apache/accumulo/core/fate/zookeeper/FateLock.java

```diff
@@ -106,14 +91,8 @@ public FateId getFateId() {
       return fateId;
     }
 
-    public byte[] serialize() {
-      byte[] typeBytes = lockType.name().getBytes(UTF_8);
-      byte[] fateIdBytes = fateId.canonical().getBytes(UTF_8);
-      byte[] result = new byte[fateIdBytes.length + 1 + typeBytes.length];
-      System.arraycopy(typeBytes, 0, result, 0, typeBytes.length);
-      result[typeBytes.length] = ':';
-      System.arraycopy(fateIdBytes, 0, result, typeBytes.length + 1, fateIdBytes.length);
-      return result;
+    public String serialize() {
+      return lockType.name() + ":" + fateId.canonical();
```

Comment:
This is SO much nicer, not having to do all that manual effort to copy bytes around is great.